pkg/stores/sqlproxy/proxy_store.go

```diff
@@ -58,7 +58,16 @@ var (
 		"_v1_Pod":       {{`spec`, `containers`, `image`}, {`spec`, `nodeName`}},
 		"_v1_ConfigMap": {{`metadata`, `labels[harvesterhci.io/cloud-init-template]`}},
 
-		"management.cattle.io_v3_Node": {{`status`, `nodeName`}},
+		"cluster.x-k8s.io_v1beta1_Machine":     {{`spec`, `clusterName`}},
+		"management.cattle.io_v3_Node":         {{`status`, `nodeName`}},
+		"management.cattle.io_v3_NodePool":     {{`spec`, `clusterName`}},
+		"management.cattle.io_v3_NodeTemplate": {{`spec`, `clusterName`}},
+		"provisioning.cattle.io_Cluster": {
```

Comment:
AFAIK this needs a group as well
```suggestion
		"provisioning.cattle.io_v1_Cluster": {
```